source/Paths.hx

```diff
@@ -139,54 +136,37 @@ class Paths
 	}
 
 	static public function getLibraryPath(file:String, library = "preload")
-	{
 		return if (library == "preload" || library == "default") getPreloadPath(file); else getLibraryPathForce(file, library);
-	}
 
 	inline static function getLibraryPathForce(file:String, library:String, ?level:String)
```

Comment:
make sure to add the return types for some of these, otherwise it'll throw errors, cuz the compiler will auto assign an type to it